packages/utilities/src/utilities/smoothScrolling/smoothScrolling.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Handle OnClick
+ *
+ * @param {*} e event object
+ */
+const smoothScrolling = e => {
+  e.preventDefault();
+  const id = e.currentTarget.getAttribute('href');
+  document.querySelector(id).scrollIntoView({
+    behavior: 'smooth',
+    block: 'start',
+  });
+};
```

Comment:
Are we going to update `TableOfContents.js`, or save that for another pull request?

Reply:
save for another full request.